### ISSUE TITLE
Keep running into dependency issues

### ISSUE THREAD

**Issue by Luthien-in-edhil** [xchwarze/SMF2WPBridge#6]
Hello,

I installed the WP2SMFBridge plugin on Wordpress and that worked brilliantly! Very simple and elegant, thank you! :)

Today I thought I'd also install the `SMF2WPBridge` plugin on SMF. However, this is more problematic: for some reason `Subs-SMF2WP-User.php` keeps running into dependency issues. 
After installing the mod in SMF, I tried logging in with a user created on both sides (SMF & WP). The screen went blank. 
Then I logged in as the SWF admin to check if the passwords were equal (not sure if that is required though, but just to make sure) and edited that SWF user to have the same password as it has on WP. 

Doing this called this function on line 63 of `Subs-SMF2WP-User.php`:

``` php
function smf2wp_integrate_reset_pass($memberName, $memberName, $password){
```

and it runs into a problem on line 70: 

``` php
$user = username_exists($memberName);
```

because it cannot find the username_exists function located in `[WP]/wp_includes/user.php`
I tried adding that with a require statement, but this leads into an endless chain of other requirements - I had a look if I could find how make all those dependencies available at once, but it got me nowhere so far. Maybe I just forgot to set something, the path to the Wordpress installation was OK (simply '../' ) - my directory layout is like this: 
- [WP_root]
- wp_includes
  - admin-bar.php
  - atomlib.php
  - author-template.php
  - ...
- wp_content
  - index.php
  - languages
  - index.php
  - ...
- forum [SMF_root]
  - Sources
    - Admin.php
  - BoardIndex.php
  - ...
  - Packages
  - Settings.php
  - ... etcetera

So I think that is how you stated that it should be able to work. 

Any ideas what might be causing this? 
Thanks again, especially the WP2SMF plugin was exactly what I needed!

**Comment by pyksid** [comment]
Hello @Luthien-in-edhil,

I don't know why but it worked before, maybe a WordPress update. I have used SMF2WP in my previous website and it worked perfectly. 

If you want to SMF2WP to work again, you need to delete this line in Subs-SMF2WP-User.php : 

`define('SHORTINIT', true);`

This allow WordPress to load all his components. It is slower but it works.

**Comment by xchwarze** [comment]
Alredy fixed in v1.1.1
